infra/test/setup/main.tf

```diff
@@ -27,6 +27,7 @@ module "project" {
   activate_apis = [
```

Comment:
future: the apis listed here should include everything needed to deploy into a project. This has the secondary benefit of making the separation of responsibilities between setup and terraform module more similar to the separation between console and terraform.

In terraform-dynamic-python-webapp, we have all project default services and all app-specific requirements enabled in setup, and then again inside the terraform so that terraform apply can continue to work on its own. When CI runs, it uses enable_apis=false to disable redundant API enablement.